assets/src/components/reader-themes-context-provider/index.js

```diff
@@ -64,14 +80,19 @@ export function ReaderThemesContextProvider( { wpAjaxUrl, children, currentTheme
 	const [ downloadingThemeError, setDownloadingThemeError ] = useState( null );
 
 	/**
-	 * If the currently selected theme is unavailable and not installable, unset the reader theme option. This will handle cases where
-	 * the reader theme stored in the options is removed and can no longer be installed.
+	 * If the currently selected theme is unavailable and not installable, or the current theme is the active theme,
+	 * unset the reader theme option.
 	 */
 	useEffect( () => {
-		if ( selectedTheme.availability === 'non-installable' ) {
-			updateOptions( { reader_theme: null } );
+		if ( themeWasOverridden ) { // Only do this once.
+			return;
+		}
+
+		if ( selectedTheme.availability === 'non-installable' || originalSelectedTheme.availability === 'active' ) {
+			updateOptions( { reader_theme: 'legacy' } );
```

Comment:
I'm reconsidering if we should force the user to re-select a theme. Defaulting to the legacy theme could imply that it's being recommended, which is not what is intended.

Reply:
Yeah, that's where I was leaning earlier on, but I could go either way. If leaving it empty is preferred, this could just be set to `null`. @westonruter Would you like to cast a vote?

Reply:
Also a reminder that this affects what's selected in the onboarding wizard as well.

Reply:
Yeah, let's go with `null`.

Reply:
Or not. If it is set to `null` then the UI shows:

![image](https://user-images.githubusercontent.com/134745/89579693-15b8b600-d7e9-11ea-8984-ab6f9dda1914.png)

We'd also need to make sure that we expand the drawer so the user knows they need to select something. Otherwise, there's no indication for how they can proceed.

We'll leave `null` as the behavior in the onboarding wizard only for initial onboarding because they must select something in each step.